man/rules/meson.build

```diff
@@ -992,6 +992,7 @@ manpages = [
    'systemd-journald@.service',
    'systemd-journald@.socket'],
   ''],
+ ['systemd-keyutil', '1', [], ''],
```

Comment:
hmm, there's the keyutils package which is quite different in purpose... keytool is already taken by java. keyctl is already a tool from keyutils.

hence I suggested `systemd-keymgmt`, to not conflict with either. systemd-keymngr might work too, systemd-keyadm too?,

Reply:
@poettering keyutils ships `keyctl`, not `keyutil`, so `keyutil` is free property if you ask me, especially if we prefix it with `systemd-`. management or manager imply too much that we manage keys which the tool doesn't.